### PR TITLE
node: Add accepted formats and examples for rpc and ws parameter desc

### PR DESCRIPTION
**Pull request by 0xluffyb**
PR for https://github.com/wormhole-foundation/wormhole/issues/3612

### CODE REVIEW

File: node/cmd/guardiand/node.go [hunk 1]
```diff
@@ -244,130 +244,129 @@ func init() {
 	guardianKeyPath = NodeCmd.Flags().String("guardianKey", "", "Path to guardian key (required)")
 	solanaContract = NodeCmd.Flags().String("solanaContract", "", "Address of the Solana program (required)")
 
-	ethRPC = NodeCmd.Flags().String("ethRPC", "", "Ethereum RPC URL")
+	ethRPC = NodeCmd.Flags().String("ethRPC", "", "Ethereum RPC URL.\nFormat: 'ws://'. Example: 'ws://eth-devnet:8545'")
```
Comment (panoel) [inline node/cmd/guardiand/node.go:247]: Does it matter if the format is 'ws://' or 'wss://'?  If we only show 'ws://' are we implying 'wss://' is not allowed?  For one testnet guardian 'wss://' was used here.
This comment applies everywhere.

**Review by panoel — APPROVED** [review]

**Comment by bruce-riley** [comment]
Will there be a follow up to this PR? Because I think we should have some verification to enforce that the parameter is the right format (with ws/wss vs. http/https, etc).

File: node/cmd/guardiand/node.go [hunk 2]
```diff
@@ -244,130 +244,129 @@ func init() {
...
 	algorandAppID = NodeCmd.Flags().Uint64("algorandAppID", 0, "Algorand app id")
 
-	nearRPC = NodeCmd.Flags().String("nearRPC", "", "near RPC URL")
-	nearContract = NodeCmd.Flags().String("nearContract", "", "near contract")
-
-	wormchainURL = NodeCmd.Flags().String("wormchainURL", "", "wormhole-chain gRPC URL")
+	nearRPC = NodeCmd.Flags().String("nearRPC", "", "Near RPC URL.\nFormat: HTTP (http://) or HTTPS (https://). Example: 'http://near:3030'")
+	nearContract = NodeCmd.Flags().String("nearContract", "", "Near contract")
 
+	wormchainURL = NodeCmd.Flags().String("wormchainURL", "", "Wormhole-chain gRPC URL.\nFormat: `<host>:<port>`. Example: 'wormchain:9090'")
```
Comment (panoel) [inline node/cmd/guardiand/node.go:315]: Is this correct?  It doesn't have an http or ws prefix?  It is IP:port?

Comment (bruce-riley) [reply node/cmd/guardiand/node.go:315]: Yes, this is correct. The GRPC library doesn't expect the prefix.

**Review by bruce-riley — APPROVED** [review]

**Pull request merged**